README.md

```diff
@@ -55,6 +60,98 @@ You can start editing the page by modifying `pages/index.js`. The page auto-upda
 
 The `pages/api` directory is mapped to `/api/*`. Files in this directory are treated as [API routes](https://nextjs.org/docs/api-routes/introduction) instead of React pages.
 
+## Adding remote repositories
+
+Notes:
+
+- For devfile hosts (environment variable or config file) specify a remote url or a local folder structure and `index.json` that follows the `devfile/api` spec.
+- You can have infinitely many sources as long as the name for the source repository is different.
+- There are two types of source types, url and stacks.
+  - url is for specifying a remote hosts.
+  - stacks is for specifying a local hosts. NOTE: Stacks MUST have `index.json` under the root path specified, an example is below.
+
+`/stacks`
+
+```
++-- stacks
+|   +-- java-maven
+|       +-- devfile.yaml
+|   +-- java-openliberty
+|       +-- devfile.yaml
+|   +-- index.json
+```
+
+`/config/index.json`
```

Comment:
can you pls add this file to .gitignore

```
MacBook-Pro:registry-viewer maysun$ git status
On branch update-for-prod
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
	modified:   config/devfile-registry-hosts.json


```

Reply:
I cant do that. Technically this is an typo in the docs that I just fixed because the `index.json` is only under the devfile folder.

Reply:
oops wrong line, i meant `/config/devfile-registry-hosts.json`